rampwf/utils/submission.py

```diff
@@ -355,22 +356,21 @@ def bag_submissions(problem, cv, y_train, y_test, predictions_valid_list,
 
     df_scores = pd.concat({step: pd.DataFrame(scores)
                            for step, scores in bagged_scores.items()})
-    df_scores.columns = df_scores.columns.rename('scores')
+    df_scores.columns = df_scores.columns.rename('score')
     df_scores.index = df_scores.index.rename(['step', 'n_bag'])
+    # bagging learning curves can be plotted on this df_scores
 
     # prepare the bagged scores which will be printed.
-    numerical_precision = pd.Series([score.precision for score in score_type],
-                                    index=[score.name for score in score_type])
     highest_level = df_scores.index.get_level_values('n_bag').max()
-    df_scores_rounded = df_scores.loc[(slice(None), highest_level), :].round(
-        numerical_precision
-    )
-    df_scores_rounded.index = df_scores_rounded.index.droplevel('n_bag')
-    print_df_scores(df_scores_rounded, score_type, indent='\t')
+    df_scores = df_scores.loc[(slice(None), highest_level), :]
```

Comment:
You should not do that

df_scores corresponds to the dataframe which will be dumped on the disk. Therefore, it contains all bags and all scores information.

df_scores_rounded takes only the last bag, round it and print it.

Since the saving is coming after the printing, we do not store all the information which are expected by the database.

Reply:
OK.